extensions/amp-story/1.0/amp-story-affiliate-link.css

```diff
@@ -0,0 +1,175 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ [data-affiliate-link-icon] {
```

Comment:
`data-` attributes are valid AMP by default, which means that publishers could start using this feature right away. Could we remove the `data-` prefix, so we get better control on how and when we want to launch this, by making the attribute valid AMP once we're ready?